### DISCUSSION TITLE
Render.com deploy error: React alpha not found

### DISCUSSION THREAD

**Discussion by Kay2dan** [blitz-js/blitz#3114] [Q&A]
Hi, I am getting the following error when I try to deploy to Render.com. Has anyone come across a similar issue before?

> error Couldn't find any versions for "react" that matches "alpha"

I am on Blitz v0.38.6

**Comment by Kay2dan** [comment]
I am not sure where the version `alpha` came from within my package.json file, however, once I renamed the field value to the conventional React versions, as within Blitz.js package.json repo, it all appears to be fine.

**Reply by beerose** [reply]
I think there's no alpha tag anymore. In the newest Blitz versions is should be fine, because React is pinned to 18 beta.

**Reply by Kay2dan** [reply]
Thank you, yes, I had to change it to get it working.
